wingtips-core/src/main/java/com/nike/wingtips/Span.java

```diff
@@ -78,9 +62,10 @@
     private final SpanPurpose spanPurpose;
     private final long spanStartTimeEpochMicros;
     private final long spanStartTimeNanos;
-
+    
     private Long durationNanos;
-
+    private Map<String,String> tags = new HashMap<String,String>(0);
```

Comment:
usually linkedhashmap is nicer as doesn't cost much yet retains order which is nicer in tests

Reply:
Agreed.